tasks/legacy_urls.md

```diff
@@ -0,0 +1,57 @@
+# GCI Tasks: Legacy URLs migration
+
+## Prerequisites
+
+* A working Zulip development environment. See
+  [here](https://github.com/zulip/zulip-gci/blob/master/README.md) for instructions
+  on how to set one up.
+* Experience working in the Zulip server environment.  We recommend
+completing the
+[Intro to Zulip Server development beginner task](https://github.com/zulip/zulip-gci/blob/master/tasks/intro-to-zulip-server.md)
+before claiming a Legacy URLs migration task, since this will require
+the knowledge you learn with that task.
+
+## Background
+
+Zulip has a number of "legacy" API URLs that are not properly
+available via the Zulip REST API, which are present in
+`zproject/legacy_urls.py` in the main Zulip project.
+
+Please carefully read [https://github.com/zulip/zulip/issues/611](the
+master GitHub issue for this migration) for details on the migration
```

Comment:
The link and the description are reversed here, should be:
[the master GitHub issue for this migration](https://github.com/zulip/zulip/issues/611)